source/github-helpers/index.ts

```diff
@@ -1,15 +1,14 @@
 import select from 'select-dom';
 import oneTime from 'onetime';
 import compareVersions from 'tiny-version-compare';
-import {isPR, isIssue, utils} from 'github-url-detection';
+import detection from 'github-url-detection';
```

Comment:
```suggestion
import urlDetection from 'github-url-detection';
```

Just `detection` is too general.